File: tools/benchmarking/fibonacci/benchmark.js
function fib(n) {
    if (n < 2) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

function main() {
    const N = 42;
    const result = fib(N);
    console.log(`JS      | Resultado: ${result}`);
}

main();

File: tools/benchmarking/fibonacci/benchmark.go
package main

import "fmt"

func fib(n int) int {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	const N = 42
	result := fib(N)
	fmt.Printf("Go      | Resultado: %d\n", result)
}
